.github/workflows/test-publish.yml

```diff
@@ -71,6 +71,12 @@ jobs:
         with:
           name: it-tests
           path: it-tests.zip
+      - name: Publish tests reports
+        if: success() || failure()
```

Comment:
or `always()` ?

Reply:
you're right, the task won't executed if cancelled